bigip/resource_bigip_ltm_virtual_server.go

```diff
@@ -401,6 +413,9 @@ func resourceBigipLtmVirtualServerUpdate(d *schema.ResourceData, meta interface{
 		TranslateAddress: d.Get("translate_address").(string),
 		VlansEnabled:     d.Get("vlans_enabled").(bool),
 	}
+	if d.Get("state").(string) == "disabled" {
+		vs.Disabled = true
```

Comment:
shouldn't this be the equivalent to the Read logic only in reverse? and handle both enabled and disabled states? e.g.

```
	if d.Get("state").(string) == "enabled" {
		vs.Enabled = true
	else {
		vs.Enabled = false
	}
```

I'm also a bit confused about why two properties are needed to reflect a single state? isn't `vs.Enabled = false` already means **disabled**?

Reply:
Hi @dannyk81 ,
BIG-IP accepts either `enabled:true` or `disabled:true` for State change from Enable to Disable.
After verifying this Manually decided to go this logic